dmoj/checkers/linematches.py

```diff
@@ -0,0 +1,31 @@
+from re import split as resplit
+from typing import List, Union
+
+from dmoj.error import InternalError
+from dmoj.result import CheckerResult
+from dmoj.utils.unicode import utf8bytes
+
+
+def check(
+    process_output: bytes, judge_output: bytes, point_value: float = 1,
+    point_distribution: List[int] = [1], filler_lines_required: bool = True, **kwargs
+) -> Union[CheckerResult, bool]:
+    process_lines = list(filter(None, resplit(b'[\r\n]', utf8bytes(process_output))))
+    judge_lines = list(filter(None, resplit(b'[\r\n]', utf8bytes(judge_output))))
+
+    if len(judge_lines) != len(point_distribution):
```

Comment:
Nit: we can do this before wasting time processing the process output.

Reply:
Micronit: usually our IEs start with lowercase letters (though there are a few exceptions).

Reply:
Resolved